element_array_ephys/readers/openephys.py

```diff
@@ -31,13 +31,13 @@
 class OpenEphys:
 
     def __init__(self, experiment_dir):
```

Comment:
Smart to do this in the class. I instead made adjustments [on the call](https://github.com/datajoint/element-array-ephys/pull/41/files#diff-3a2e32e7f77959667b86eb13f37a6fffbce93cac4e75e8776ebee7ba8f7123b3R197). I like this solution better

Reply:
Hi @CBroz1, unless I am mistaken, I think we are actually implementing this the same way.

Reply:
No, sorry, I was mistaken. We both switched to using the full path within the make function.